### PR TITLE
org.wso2.carbon.policyeditor.ui code refactored

### PR DESCRIPTION
**Pull request by cdwijayarathna**
Signed-off-by: cdwijayarathna <cdwijayarathna@gmail.com>

### CODE REVIEW

File: components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties [hunk 1]
```diff
@@ -1,3 +1,21 @@
+#
+# Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
```
Comment (mohanvive) [inline components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties]: Fix the year in license header..

File: components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/Resources.properties [hunk 2]
```diff
@@ -1,3 +1,21 @@
+#
+# Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (mohanvive) [inline components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/Resources.properties]: Fix the year.

File: components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/web/policyeditor/css/local-styles.css [hunk 3]
```diff
@@ -1,16 +1,19 @@
-/* Copyright 2006-2008 WSO2, Inc. http://www.wso2.org
+/*
+ * Copyright (c) 2008, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (mohanvive) [inline components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/web/policyeditor/css/local-styles.css]: Fix the year..

File: components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties (hunk 1 above)
Comment (cdwijayarathna) [inline components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties]: This file has been created and added in 2014, should I change year? if so what should I use?

File: components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/Resources.properties (hunk 2 above)
Comment (cdwijayarathna) [inline components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/Resources.properties]: This file has been created and added in 2014, should I change year? if so what should I use?

File: components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/web/policyeditor/css/local-styles.css (hunk 3 above)
Comment (cdwijayarathna) [inline components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/web/policyeditor/css/local-styles.css]: This file has been created and added in 2008, should I change year? if so what should I use?

File: components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties (hunk 1 above)
Comment (mohanvive) [inline components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties]: No, actualy speaking above file is not created in 2014. This file is available in 4.0.0 branch as well (which means in  2013).. It seems, you have added a license header for these files. IMO, If you are not aware about the exact year that file created. Then better to add current year when u added license header..

Comment (cdwijayarathna) [inline components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties]: Thanks, I'll double check with team about the exact year this file was created and if fail to find exact year replace with this 2015 which was the year I added license header.

**Pull request merged**